src/components/views/rooms/RoomHeader.js

```diff
@@ -117,6 +119,14 @@ export default class RoomHeader extends React.Component {
         return !(currentPinEvent.getContent().pinned && currentPinEvent.getContent().pinned.length <= 0);
     }
 
+    _displayInfoDialogAboutScreensharing() {
+        Modal.createDialog(InfoDialog, {
+            title: _t("Screensharing has changed"),
+            description: _t("You don't have to shift-click anymore! You can now share " +
+            "your screen in any video call and in voice calls if other side supports it."),
```

Comment:
I'll let design weigh in on this too, but suggest maybe:

> ***Screen sharing is here!***
> 
> You can now share your screen by pressing the 'screen share' button during a call. You can even do this in audio calls if both sides support it!

Reply:
Yeah, sounds much better!